### PR TITLE
suite-nonsplit-cms-pages

### PR DESCRIPTION
**Pull request by gorav-sharma-metacube**
## PR Description
CC-16790: suite-nonsplit-cms-pages

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/content_endpoints/cms_pages/positive.robot [hunk 1]
```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup     TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
```
Comment (Katerynaa-Lavryk) [inline tests/api/suite_nonsplit/glue/content_endpoints/cms_pages/positive.robot:7]: You can also add a test to check returning of some number of cms-pages: ?page[limit]=10&page[offset]=0

Comment (gorav-sharma-metacube) [reply tests/api/suite_nonsplit/glue/content_endpoints/cms_pages/positive.robot:7]: fixed

**Review by Katerynaa-Lavryk — DISMISSED** [review]

File: resources/environments/environments_api_suite.json [hunk 2]
```diff
@@ -141,15 +141,6 @@
 
             },
 
-        "concrete":{
```
Comment (Katerynaa-Lavryk) [inline resources/environments/environments_api_suite.json:144]: Why do you remove this product?

Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_suite.json:144]: It is already declared twice in the file. So i have remove duplicate one.

Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_suite.json]: Done

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Review by Katerynaa-Lavryk — APPROVED** [review]

**Pull request merged**